### PR TITLE
website: update website hero and description

### PR DESCRIPTION
**Pull request by slemeur**
### What does this PR do?

Update the website hero and description

### What issues does this PR fix or reference?

[](https://github.com/containers/podman-desktop/issues/8873)

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -106,11 +106,12 @@ function Hero(): JSX.Element {
         <div className="text-center lg:w-2/3 w-full bg-hero-pattern bg-no-repeat bg-center">
           <div className="bg-white/30 dark:bg-transparent">
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-8 font-medium text-gray-900 dark:text-white leading-[1.2]">
-              Containers and Kubernetes for application developers
+              Best Free & Open Source Tool for Containers & Kubernetes - Podman Desktop
```
Comment (feloy) [inline website/src/pages/index.tsx]: I suppose you need to escape these characters
```suggestion
              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes - Podman Desktop
```

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -106,11 +106,12 @@ function Hero(): JSX.Element {
...
             </h1>
             <p className="text-base md:text-lg">
-              Podman Desktop is an open source graphical tool enabling you to seamlessly work with containers and
-              Kubernetes from your local environment.
+              Podman Desktop is the best free and open source tool to work with Containers and Kubernetes for developers.
+              Get an intuitive and user-friendly interface to effortlessly build, manage, and deploy containers and Kubernetes — all from your desktop.
```
Comment (feloy) [inline website/src/pages/index.tsx]: the `and Kubernetes` is not clear. It applies well with _manage_, but not with _deploy_

**Comment by axel7083** [comment]
What would you think about re-using some styling I made for the AI Lab page

## Current

Screenshot from 2024-09-13 10-23-56

## Gradient color

image

The colors are configurable. I also feel like the `- Podman Desktop` could be removed at the end, at it is already in the name of the website

**Comment by slemeur** [comment]
Go for it ! Good idea with the gradient!

File: website/src/pages/index.tsx (hunk 2 above)
Comment (slemeur) [reply website/src/pages/index.tsx]: What do you think about:
"Get an intuitive and user-friendly interface to effortlessly bridge the gap between local development and Kubernetes clusters. Build, manage, and deploy with ease from a single tool."

**Review by benoitf — COMMENTED** [review]
I would not mix styling and text content at the same time

**Comment by axel7083** [comment]
> I would not mix styling and text content at the same time

You don't like the gradient ?

File: website/src/pages/index.tsx (hunk 2 above)
Comment (feloy) [reply website/src/pages/index.tsx]: What about finishing with:
Build, manage, and deploy everywhere with ease from a single tool

**Comment by benoitf** [comment]
during a review, if you mix things, then you need a higher consensus:
both message and style would need to be approved while maybe all people may agree only on one topic


and on a side note, yes I dislike gradient title. To me it's part of the past.

**Comment by axel7083** [comment]
> during a review, if you mix things, then you need a higher consensus: both message and style would need to be approved while maybe all people may agree only on one topic
> 
> and on a side note, yes I dislike gradient title. To me it's part of the past.

OH yeah sorry I forgot, the styling will not be included in the PR, I said stevan that it could be in a follow up issue where the discussion could occur

**Review by feloy — APPROVED** [review]

**Comment by benoitf** [comment]
on my end the rendering is the following:
image

the - is at the end of the sentence and I'm not sure the text should be that long.

**Pull request merged**